tests/fsharp/TypeProviderTests.fs

```diff
@@ -31,13 +31,16 @@ let FSC_BASIC = FSC_OPT_PLUS_DEBUG
 let FSI_BASIC = FSI_FILE
 #endif
 
+let inline getTestsDirectory dir = __SOURCE_DIRECTORY__ ++ dir
+let testConfig' = getTestsDirectory >> testConfig
```

Comment:
I feel it would be better to hide the other `testConfig` rather than making a "prime":
* minimizes the changes
* the underlying `testConfig` is not used anywhere

I stumbled on this change while rebasing a PR where I add a test in this file, it would have just worked using the F# feature of hidding symbols.